english/love/again/again

```diff
@@ -0,0 +1,4 @@
+What came before is done and gone, can never be again.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.